Sentinel/Classes/CrashDetection/Internal/CrashManager.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  CrashManager.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 10.03.2025..
+//
+
+// Inspiration link: https://github.com/DebugSwift/DebugSwift
+
+import Foundation
+
+enum CrashManager {
```

Comment:
Based on the functionality, I would say this is more like a crash file/data manager, and if so, I would rather change the name to `CrashDataManager` or `CrashFileManager` because I though this does more than file handling just by the name.